packages/storage/src/implementation/url.ts

```diff
@@ -18,11 +18,16 @@
 /**
  * @fileoverview Functions to create and manipulate URLs for the server API.
  */
-import { DEFAULT_HOST } from './constants';
 import { UrlParams } from './requestinfo';
 
-export function makeUrl(urlPart: string): string {
-  return `https://${DEFAULT_HOST}/v0${urlPart}`;
+export function makeUrl(urlPart: string, host: string): string {
+  const protocolMatch = host.match(/^(\w+):\/\/.+/);
```

Comment:
nit/optional, I think using replace with callback would save some lines and be more readable

Reply:
I couldn't figure out how to easily set up a regex that would call the callback if the protocol wasn't there so I left it alone for now. We can always optimize any time so let me know if you have a snippet you want to put in.